app/include/zmk/keymap.h

```diff
@@ -18,7 +20,7 @@ int zmk_keymap_layer_toggle(uint8_t layer);
 int zmk_keymap_layer_to(uint8_t layer);
 const char *zmk_keymap_layer_label(uint8_t layer);
 
-int zmk_keymap_position_state_changed(uint32_t position, bool pressed, int64_t timestamp);
+int zmk_keymap_position_state_changed(const struct zmk_position_state_changed *pos_ev);
```

Comment:
example of a function signature that would become more complex if we'd need to keep a `zmk_event_t` around instead of `zmk_position_state_changed`.